integrations/kube-agent-updater/cmd/teleport-kube-agent-updater/main.go

```diff
@@ -136,8 +137,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	statefulsetController := controller.StatefulSetVersionUpdater{
```

Comment:
Do we need to start both (deployment and sset) always?

Reply:
They only work when they are invoked following a resource-related event. This is purely reaction-based, the absence of resources will cause the controllers to never run. As it does not give additional cost, it's easier to always start both than trying to guess/configure and have another failure mode (watching the wrong Kind).